### PR TITLE
CAL-308 Updated NSILI mapping of subjectCategoryCode

### PR DESCRIPTION
**Pull request by beyelerb**
#### What does this PR do?
Changes the NSIL Transformer to map COMMON.subjectCategoryCode to the isr.nato-reporting-code
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@dcruver @troymohl 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
#### How should this be tested?
Verify that the value(s) from subjectCategoryCode are set in isr.nato-reporting-code
(don't have test case available, verify the unit test and expected results and that it passes)
#### What are the relevant tickets?
CAL-308 (https://codice.atlassian.net/browse/CAL-308)
#### Checklist:
- [x] Documentation Updated
	- [ ] Change Log Updated
- [X] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by troymohl** [comment]
I think this should probably also handle the reverse mapping in `ResultDAGConverter`

File: catalog/nsili/catalog-nsili-transformer/src/test/java/org/codice/alliance/nsili/transformer/DAGConverterTest.java [hunk 1]
```diff
@@ -489,10 +494,13 @@ private void checkCommonAttributes(MetacardImpl metacard) {
         assertThat(stanagSourceAttr.getValue()
                 .toString(), is(COM_SOURCE));
 
-        Attribute subjCatTgtAttr = metacard.getAttribute(Isr.TARGET_CATEGORY_CODE);
+        ///Attribute subjCatTgtAttr = metacard.getAttribute(Isr.TARGET_CATEGORY_CODE);
```
Comment (jlcsmith) [inline catalog/nsili/catalog-nsili-transformer/src/test/java/org/codice/alliance/nsili/transformer/DAGConverterTest.java]: remove comment

**Comment by jlcsmith** [comment]
Agree with Troy that the outbound side should be handled as well.  Otherwise looks good

**Comment by beyelerb** [comment]
Deleting this PR and submitting another after code-formatting and addressing review comments.

**Pull request closed without merging**